bBridgeAPISDK/ImageProcessing/Structs/ObjectDetectionData.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace bBridgeAPISDK.ImageProcessing.Structs
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class ObjectDetectionData
+    {
+        #region Constructors
+        
+        /// <summary>
+        /// Create imput structure for Image Object Detection
+        /// </summary>
+        /// <param name="imageUri">Uri of the image to detect from</param>
+        /// <param name="treshold">Object detection treshold 0.01 - 1 </param>
```

Comment:
Why must `threshold` be greater than 0.01? It is weird.
`threshold` value is supposed to be in `[0; 1]`